### DISCUSSION TITLE
AI Role: Interface Improvements & Slight Buff Idea.

### DISCUSSION THREAD

**Discussion by TheDoktorZ** [unitystation/unitystation#7398] [Ideas]
Currently the AI of the station can't do more than open/close or lock/unlock doors. I have some suggestions that would make the AI way more useful, not just more powerful, but truly useful to benefit everyone on the station.

### Character Information
Currently all the AI knows about the crew is their name if they don't take notes all the time when people join the round and the game announces their name and role. Give the AI an option to activate a similar nameplate like the admins' meta-vision, the one with the nameplates under the players' characters, let the AI see their RP name, role and maybe health status (such as Alive, Alive but vacant or Dead) under them. Players can already check each other when they stand close to each other, why can't AI through cameras?

### Remote Terminal Access
Let AI access terminals, such as coms, security and medical (does the medical console even work for humans?) remotely. This is actually not a new thing, if someone leaves an ID card in the consoles AI can already access them now, but what if AI could get these to their UI? The AI could help a lot with the administration of the station (on servers where people actually care about RP).

I hope You folk like my ideas, they don't make the AI overpowered or anything, just more useful to benefit everyone.

**Comment by PerfectTangent** [comment]
converting to discussion. Also, the second idea is simply just an unimplemented feature.
